openedx/features/survey_report/queries.py

```diff
@@ -0,0 +1,58 @@
+"""
+Queries to get data from database.
+"""
+
+from datetime import datetime, timedelta
+
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+from django.db.models import Count, OuterRef, Q, Subquery
+
+from common.djangoapps.student.models import CourseEnrollment
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def get_unique_courses_offered() -> int:
+    """
+    Get total number of unique courses offered.
+    """
+    return CourseOverview.objects.annotate(
+        count=Subquery(
+            CourseEnrollment.objects
+                            .filter(course_id=OuterRef('id'))
+                            .values('course_id')
+                            .annotate(count=Count('course_id'))
+                            .values('count')
+        ))\
+        .filter(count__gt=5)\
+        .filter(start__lt=datetime.now())\
+        .filter(Q(end__isnull=True) | Q(end__gt=datetime.now()))\
+        .count()
+
+
+def get_currently_learners() -> int:
```

Comment:
This name is a bit confusing.

Reply:
@ormsbee what do you think that is a good option? Maybe something like "get_active_users_in_the_last_weeks" or is too long?

Reply:
@ormsbee Done